grails-app/migrations/join_dets_and_rxrs.groovy

```diff
@@ -272,4 +272,57 @@ databaseChangeLog = {
                    viewName: 'invalid_detection'
         )
     }
+
+    changeSet(author: "jburgess", id: "1430268900000-09") {
+        createView("""
+            SELECT
+              station_station_name AS station,
+              installation_name AS installation,
+              rxr_project_name AS project,
+              aatams.scramblepoint(st_makepoint(station_longitude, station_latitude)) AS public_location,
+              st_x(aatams.scramblepoint(st_makepoint(station_longitude, station_latitude))) AS public_lon,
+              st_y(aatams.scramblepoint(st_makepoint(station_longitude, station_latitude))) AS public_lat,
+              ('http://localhost:8080/aatams/installationStation/show/'::text || station_id) AS installation_station_url,
```

Comment:
Ah, I see that it's maintained the existing behaviour. I've added a ticket on the board for us to check it. If it is broken we can look into it.